### PR TITLE
Initialize mtail formula + import mtail program for Postfix

### PR DESCRIPTION
**Pull request by tacerus**

### CODE REVIEW

File: mtail-formula/mtail/init.sls [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - name: mtail 
+
+{%- set sysconfig = [] %}
+{%- for key, value in config['sysconfig']['args'].items() %}
+{%- if value is string %}
+{%- do sysconfig.append('-' ~ key ~ ' ' ~ value) %}
+{%- elif value is sameas true %}
+{%- do sysconfig.append('-' ~ key) %} %}
+{%- else %}
+{%- do salt.log.error('mtail: illegal sysconfig value') %}
```
Comment (cboltz) [inline mtail-formula/mtail/init.sls:32]: I'd recommend to also allow / check for `false` before erroring out

Comment (tacerus) [reply mtail-formula/mtail/init.sls:32]: Good point!

File: mtail-formula/mtail/init.sls [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - require:
+      - pkg: mtail_package
+
+{%- set programs_directory = config['sysconfig']['args']['progs'] %}
+mtail_programs_directory:
+  file.directory: {#- TODO: submit directory installation to the package #}
+    - name: {{ programs_directory }}
+    - group: mtail
+    - require:
+      - pkg: mtail_package
```
Comment (tacerus) [inline mtail-formula/mtail/init.sls:50]: mtail already ships /etc/mtail, unsure how this came to be here.

File: mtail-formula/mtail/init.sls [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      - file: mtail_programs_directory
+
+{#- TODO: repair packaged service to correctly implement "reload" #}
+mtail_reload:
+  cmd.run:
+    - name: systemctl is-active mtail && systemctl kill --signal=SIGHUP mtail || true
+    - onchanges:
+      - file: mtail_programs
+    - require:
+      - pkg: mtail_package
```
[the remark this answers is not in the document]
Comment (tacerus) [reply mtail-formula/mtail/init.sls:74]: https://build.opensuse.org/request/show/1154012

File: mtail-formula/mtail/init.sls (hunk 2 above)
Comment (tacerus) [reply mtail-formula/mtail/init.sls]: Likely to facilitate non-default program directories, however no need for our use case.

**Comment by tacerus** [comment]
@cboltz ready when you are

**Comment by tacerus** [comment]
Actually, I should probably patch this to our new `suse_sysconfig.header` now.

**Comment by tacerus** [comment]
Done, much cleaner. :)

**Pull request merged**